manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -37,6 +37,9 @@ spec:
         service.istio.io/canonical-revision: {{ .Values.revision }}
 {{- else}}
         service.istio.io/canonical-revision: latest
+{{- end }}
+{{- if not $gateway.runAsRoot }}
+        istio.io/unprivileged-pod: "true"
```

Comment:
Any reason to use a label rather than the more common `ISTIO_META_UNPRIVILEGED_POD` to pass via node metadata?

Reply:
hmm. that might make more sense, and it doesn't require an API change. will do that

Reply:
thanks!